datafusion/physical-expr/src/aggregate/min_max.rs

```diff
@@ -1073,24 +1116,90 @@ mod tests {
     }
 
     #[test]
-    fn min_time64() -> Result<()> {
+    fn min_time32second() -> Result<()> {
+        let a: ArrayRef = Arc::new(Time32SecondArray::from(vec![1, 2, 3, 4, 5]));
+        generic_test_op!(
+            a,
+            DataType::Time32(TimeUnit::Second),
+            Max,
+            ScalarValue::Time32Second(Some(5))
```

Comment:
I think this should be
```suggestion
            Min,
            ScalarValue::Time32Second(Some(1))
```

Same below for `min_time32millisecond` and `min_time64nanosecond` and `min_time64microsecond`.

Reply:
Thank you very much for spotting this out!